nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.listen(3000, () => console.log(`Calculator:listening on port 3000`));
+app.get("/calculator/multiply", (req, res) => {
+    const values = Object.values(req.query).flat()
+    console.log(values);
+    const multiply = values.reduce((acc, val) => acc * val)
+    
+    if(isNaN(multiply)){
```

Comment:
Nice that you thought about if the query parameters were numbers 💪

Reply:
Thanks for your valuable feedback